features/step_definitions/cli.rb

```diff
@@ -183,6 +183,16 @@
     })
 end
 
+Given /^I wait(?: up to #{NUMBER} seconds)? for the last background process to finish$/ do |seconds|
+  seconds = Integer(seconds) rescue 60
```

Comment:
@xingxingxia if you are using the `#{NUMBER}` already `seconds` should be an `Integer` already.  I think you can use ` seconds ||= 60` instead of using the `rescue` clause.

Reply:
It will be a number but still will be a String.